pbf/templates/game.html

```diff
@@ -103,6 +103,8 @@ <h3>Host Actions:</h3>
 	  <button type="submit" class="btn">Change scenario</button>
 	</form>
 
+  <a href="/api/game/{{ game.id }}">Back up game data</a>
```

Comment:
"Back up game data" is inaccurate, as it is just a link **to** the game data. The link has a verb, but it doesn't actually perform any back up; that's for the person accessing this link to manually do. I think it may be better to just say it is a link to the data, and state the it can be used to make a backup.